Makefile

```diff
@@ -157,7 +157,8 @@ diff_pylint_report: pylint_report.txt
 # statement). So we run nose inside of coverage.
 .coverage: $(PYSOURCES) $(wildcard tests/*.py) khmer/_khmermodule.so
 	coverage run --branch --source=scripts,khmer,oxli --omit=khmer/_version.py \
-		-m nose --with-xunit --attr=\!known_failing --processes=0
+		-m nose --with-xunit --attr=\!known_failing,\!huge \
```

Comment:
Go for it! I don't know what are our guidelines for cases like this, but if you don't want to commit directly to this branch you can always make a PR for other PRs. But I don't mind if you do directly =]